homeassistant/components/wink.py

```diff
@@ -97,9 +97,16 @@ def setup(hass, config):
     hass.data[DOMAIN] = {}
     hass.data[DOMAIN]['entities'] = []
     hass.data[DOMAIN]['unique_ids'] = []
+
+    def keep_alive_call(event):
+        """Call the Wink API endpoints to keep PubNub working."""
+        pywink.wink_api_fetch()
+        pywink.get_user()
+    hass.services.register(DOMAIN, 'keep_alive', keep_alive_call)
```

Comment:
I would rather not expose it, I agree. I originally was passing the keep_alive function to pubnubsubhandler which would call it every hour. I confirmed that was functioning correctly on my system, buy outputting the response of get_user() and wink_api_fetch() to a temp log file. However it apparently isn't working still for some users. It turns out there is no need to call get_user(), calling either manually will fix the problem. I made it a service so a user in the forums could test manually executing it when their states stopped working. That seems to work for them. Do you have any idea why automatically calling the function could work for some people but not others?

This is a quote from a user in the forum, prior to me making this a service.

"There was still nothing in the temp.log after an hour (and "Polling API" appeared in HA logs), so I played around with your code and changed the keep_alive_call to a service that I could call through HA. When I call that service through the front end, the temp.log file updates."

The temp.log file was setup to just output the response form the two calls to pywink. His logs were empty, and mine were not, using the exact same code.

I am thinking it might be best to move the automated calls to pywink out of pubnubsubhandler and into wink.py? Thoughts?